parachain/primitives/router/src/outbound/mod.rs

```diff
@@ -27,7 +25,7 @@ where
 	OutboundQueue: OutboundQueueTrait,
 	OutboundQueue::Ticket: Encode + Decode,
 {
-	type Ticket = (Vec<u8>, XcmHash);
+	type Ticket = Vec<u8>;
```

Comment:
Is removing XcmHash nessessary here? Seems is required in BridgeHubRococoOrBridgeHubWococoSwitchExporter and will cause cumulus fail to build?